src/main/java/edu/wpi/grip/ui/PaletteView.java

```diff
@@ -71,24 +71,20 @@ public PaletteView(EventBus eventBus) {
         this.eventBus.register(this);
     }
 
-
-    // XXX: This is the only way to do this unfortunately.
-    // This is where this came from: https://bitbucket.org/controlsfx/controlsfx/issues/330/making-textfieldssetupclearbuttonfield
-    private void setupClearButtonField(CustomTextField customTextField) throws Exception {
-        Method m = TextFields.class.getDeclaredMethod("setupClearButtonField", TextField.class, ObjectProperty.class);
-        m.setAccessible(true);
-        m.invoke(null, customTextField, customTextField.rightProperty());
-    }
-
     /**
-     * @param searchText The text to search for
-     * @param name       The name of the operation to check
-     * @return true if the search text matches the name
+     * Make the search box have a "clear" button
+     * <p>
+     * XXX: This is the only way to do this unfortunately.
+     * This is where this came from: https://bitbucket.org/controlsfx/controlsfx/issues/330/making-textfieldssetupclearbuttonfield
      */
-    private boolean searchAlgorithmMatches(String searchText, String name) {
-        // TODO: Make this use a fuzzy search
-        final String lowerCaseSearchString = searchText.toLowerCase();
-        return lowerCaseSearchString.isEmpty() || name.toLowerCase().contains(lowerCaseSearchString);
+    private void setupClearButtonField(CustomTextField customTextField) {
+        try {
+            final Method m = TextFields.class.getDeclaredMethod("setupClearButtonField", TextField.class, ObjectProperty.class);
+            m.setAccessible(true);
+            m.invoke(null, customTextField, customTextField.rightProperty());
+        } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
+            e.printStackTrace();
```

Comment:
Not being able to show a clear button that isn't guaranteed to be available to us isn't an illegal state IMO.  Logging a stack trace and letting the textfield otherwise keep working is a good way to gracefully handle that error.